src/PathOrderOptimizer.h

```diff
@@ -501,7 +501,19 @@ class PathOrderOptimizer
                     score += 1000; //1 meter penalty.
                 }
             }
-            if(score < best_score)
+
+            constexpr float EPSILON = 25.0;
+            if(best_score - EPSILON <= score && score <= best_score + EPSILON)
```

Comment:
```suggestion
            if(score >= best_score - EPSILON  && score <= best_score + EPSILON)
```
or something like 
`if (fabs(score - best_score) <= EPSILON)`
would be a bit clearer

Reply:
I see where you are coming from, it is a lot easier to read when I think about it in that format 👌

Reply:
https://github.com/Ultimaker/CuraEngine/pull/1623/commits/fcf4eac5c4177f0d47b43383bf8e91db7b4cad70